modules/generic_worker/manifests/control_bug.pp

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This class is used to create and close bugs automatically 
```

Comment:
_This class provides a utility to the worker, to enable it to raise a Bugzilla bug against itself, if it considers that it is in a bad state._